index.js

```diff
@@ -45,8 +46,26 @@ function listContacts(contacts) {
 }
 
 function sortByFirstName() {
```

Comment:
Same feedback here as for `sortByFirstName`. Separating out your click events and DOM manipulation from this pure function will help make your code more predictable in the long run.